static/absolute/js/searchPage.js

```diff
@@ -147,57 +157,211 @@ function showRestaurants(restaurantsResponse) {
     return;
   }
 
-  //This will be used to create a new div for every restaurant returned by the Places Library:
-  allRestaurants.results.forEach((restaurant) => {
-    let restaurantDiv = document.createElement("div");
-    restaurantDiv.classList.add("restaurant-card");
+  //Used to get additional information about the restaurant results.
+  const fields = "url,formatted_phone_number,website,review";
+
+  //Create a new div for every restaurant returned by the Places Library:
+  let restaurantIndex = 1;
+  for (restaurant of allRestaurants.results) {
+    //Get additional details for every restaurant using that restaurant's place id.
+    let placeDetailsResponse = await (
+      await fetch(
+        `/api/placedetails?fields=${fields}&id=${restaurant.place_id}`
+      )
+    ).json();
+
+    let additionalDetails =
```

Comment:
General note: I think it makes sense to prefer `const` by default. So all variables should be `const`, and if, you need to re-assign something, use `let`. This seems better because `const` variables tell the user that "this value doesn't change", which makes it easier to read/debug code. When I see a `let`, I assume that the value is going to be re-assigned later.

I would go through this PR and change all `let`s to `const`, if the variable declared with `let` is not being re-assigned.

Reply:
+1